keystone/common/cass/core.py

```diff
@@ -144,3 +148,19 @@ def is_secondary_idx_on_col(model_cls, column):
         return False
 
     return True
+
+class QuorumFallBackRetryPolicy(RetryPolicy):
+    def on_unavailable(self, query, consistency, required_replicas, alive_replicas, retry_num):
+        if retry_num != 0:
+            return (self.RETHROW, None)
+        else:
+            if ConsistencyLevel.LOCAL_QUORUM == consistency:
+                return (self.RETRY, ConsistencyLevel.QUORUM)
+            else:
+                return (self.RETHROW, None)
+
+def connect_to_cluster(ips, keyspace):
+    return connection.setup(ips, keyspace, consistency = ConsistencyLevel.LOCAL_QUORUM, 
```

Comment:
Since we are using a DC aware policy, somewhere here we will need to say what our "local datacenter" is. This parameter will then need to match the one of the string ids we will use to configure the NetworkTopologyAware replication. Does this make sense? What this boils down to, for this piece of code: we will need the ability to configure keystone with a string id representing its local data center.